similar/editor/med.cpp

```diff
@@ -878,10 +903,13 @@ static void close_editor()
 			break;
 
 		case 3:
-			set_screen_mode(SCREEN_GAME);		//put up game screen
-			Game_mode = GM_EDITOR;
-			editor_reset_stuff_on_level();
-			N_players = 1;
+			if (!Game_wind)	// if we're already playing a game, don't touch!
```

Comment:
Keep this as-is (unless you rebase to undo the hunk entirely), but note for next time that it is often cleaner and reduces churn to use flow control explicitly.  In this case, you could have added

```c++
    if (Game_wind)
        break;
```

immediately before the call to `set_screen_mode` to get the same effect, but avoid reindenting all subsequent lines.